app/javascript/hooks/use-theme-observer.tsx

```diff
@@ -63,11 +78,31 @@ export function useThemeObserver(): ThemeData {
         }))
       }
     }
-    mediaQueryList.addEventListener('change', mediaQueryListener)
+
+    const mediaQueryListAccessibility = window.matchMedia(
+      '(prefers-contrast: more)'
+    )
+    const mediaQueryListenerAccessibility = () => {
```

Comment:
```suggestion
    const mediaQueryListenerAccessibilityDark = () => {
```
(to be more explicit)